ui/components/app/name/name-details/index.scss

```diff
@@ -7,4 +7,9 @@
   &__address .mm-text-field {
     padding-right: 5px;
   }
+
+  .mm-modal-body {
+    // Allows the name suggestion combobox dropdown to overflow the modal body.
+    overflow-y: visible;
```

Comment:
Oh this is interesting. When we decided to adjust the scrolling behaviour of the Modal I didn't test the behavior of dropdowns but this makes sense. My one suggestion is to use the `ModalBody` `className` prop to add a custom class to override the behaviour so it's isolated to this Modal. We should strive to always add custom classnames for additional styling.

Reply:
Done. Thanks for explaining.